src/blocks/remote-data-no-results/render.php

```diff
@@ -7,10 +7,10 @@
 // $content (string): The block default content.
 // $block (WP_Block): The block instance.
 
-$should_render_empty_result = BlockBindings::should_render_empty_result( $block );
+$state = BlockBindings::is_error_or_empty_state( $block->context, $attributes );
 
-// Skip the rendering if the block's results are not empty.
-if ( ! $should_render_empty_result ) {
+// The state will only be true when the query gives back an error, or no results and the block attribute matched the response.
```

Comment:
This comment doesn't quite make sense

Reply:
I fixed this already, but didn't push up the follow up commit. Will push that up now